plugins/mac-avcapture/av-capture.mm

```diff
@@ -2396,7 +2396,7 @@ bool obs_module_load(void)
 	av_capture_info.output_flags = OBS_SOURCE_ASYNC_VIDEO |
 				       OBS_SOURCE_AUDIO |
 				       OBS_SOURCE_DO_NOT_DUPLICATE;
-	av_capture_info.get_defaults = av_capture_defaults_v2,
+	av_capture_info.get_defaults = av_capture_defaults_v2;
```

Comment:
This seems like a typo that could be fixed on its own?

Reply:
Sure, can move it into a separate PR.

Reply:
Will automatically be removed once https://github.com/obsproject/obs-studio/pull/8682 is merged and the PR will be rebased.